ethereum/test/test_verification.js

```diff
@@ -58,13 +69,16 @@ describe.skip("Verification tests", function () {
       );
       console.log(await this.lightClientBridge.latestMMRRoot());
 
-      this.channel = await BasicInboundChannel.new(this.lightClientBridge.address,
-        { from: owner }
-      );
-      this.app = await MockApp.new();
+      // this.channel = await BasicInboundChannel.new(this.lightClientBridge.address,
+      //   { from: owner }
+      // );
+      // this.app = await MockApp.new();
     });
 
     it("should successfully verify a commitment", async function () {
+      // TODO finish this test
```

Comment:
I'm assuming this TODO is intentional and will be addressed shortly?

Reply:
Yes, will come in my next PR. Thanks for the review 🙌